package/MDAnalysis/lib/util.py

```diff
@@ -2543,3 +2543,26 @@ def wrapper(self, *args, **kwargs):
                         self._kwargs[key] = arg
         return func(self, *args, **kwargs)
     return wrapper
+
+def is_installed(modulename, ignore_names=None):
+    """Checks if module is installed
+
+    Parameters
+    ----------
+    modulename : str
+        name of the module to be tested
+    ignore_names : Iterable[str]
+        list of module names to ignore when checking -- will return True
+        if modulename is in ignore_names
+    """
+    if ignore_names is None:
+        ignore_names = []
+    if modulename in ignore_names:
+        return True
```

Comment:
```suggestion
    if ignore_names is not None and modulename in ignore_names:
        return True
```

However I struggle to see the usefulness of `ignore_names`.

Reply:
it's used to distinguish absent module (like `dask`) and not a module at all (like `local`, which is the name for backend but not a valid module name)